# 客户端请求
int32 num1
int32 num2

---
#服务端响应
int32 sum